masterlist.yaml

```diff
@@ -21511,6 +21511,92 @@ plugins:
   - name: 'Holds - Falkreath.esp'
     after: [ 'Landscape Fixes For Grass Mods.esp' ]
 
+  ### HS Interior Overhauls ###
+  - name: HSDragonBridge - Four Shields Tavern.esp
```

Comment:
Plugin names should be written as
```yaml
  - name: 'Example.esp'
```
So the `'` is missing before and after the plugin names.